components/meter/meter-linear.js

```diff
@@ -1,29 +1,46 @@
 import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { bodySmallStyles } from '../typography/styles.js';
 
 class MeterLinear extends LitElement  {
 	static get properties() {
 		return {
-			value: { type: Number },
+			context: { type: String },
```

Comment:
`text` tends to be used in the other components as what is said via the screen reader. (The only example I can think of is the `d2l-card`) 

However, I think I would prefer to use `text` instead of context here. I like your suggestion for `text-inline` attribute.